src/core/fair_queue.cc

```diff
@@ -447,4 +468,21 @@ std::vector<seastar::metrics::impl::metric_definition_impl> fair_queue::metrics(
     });
 }
 
+std::vector<seastar::metrics::impl::metric_definition_impl> fair_queue::global_metrics() {
+    namespace sm = seastar::metrics;
+    return std::vector<sm::impl::metric_definition_impl>({
+            sm::make_counter("throttled_per_tick_threshold",
+                    [this] { return _throttled_per_tick_threshold; },
+                    sm::description("Number of times dispatch was throttled on the per tick threshold")),
+            sm::make_counter("throttled_no_capacity_rate",
+                    [this] { return _throttled_no_capacity_rate; },
+                    sm::description("Number of times this class was throttled dispatching requests "
```

Comment:
question: this says "this class" but as far as I can tell these are all global metrics, not class specific? However it does seem like the last two could be made class-specific easily.

Reply:
Just looking through my reflog and I had this originally be per class.

Now the question is whether I changed it for a reason or whether I just screwed something up during rebase. 🤔